agent/brandon/search.py

```diff
@@ -21,8 +25,25 @@ def _estimate_move_score(board, move):
         return (len(move.get_cells())
             + WEIGHT_SUMITO * move.is_sumito(board))
 
+    @staticmethod
+    def _is_quiescent(board):
+        """
+        Determines if the given board has adjacent opposing marbles.
+        :param board: a Board
+        """
+        for cell, color in board.enumerate_nonempty():
+            for neighbor in cell.neighbors_se():
+                neighbor_color = board[neighbor] if neighbor in board else None
+                if neighbor_color and neighbor_color != color:
+                    return False
+        return True
+
     @classmethod
-    def _order_moves(cls, board, moves):
+    def _order_moves(cls, board, moves, best_move=None):
+        if best_move:
+            # list principal variation first
+            return [best_move, *[move for move in moves if move != best_move]]
```

Comment:
not really happy with this, it'll do the listcomp/splat on basically every move enumeration which isn't ideal. Theoretically it'd be possible to just do `[best_move, *moves]` which would be more performant but it'd have a duplicate of the PV somewhere inside the move order -- the idea is that since we'd already have a transposition table entry for it and it'd effectively just get skipped. It'd be a really pragmatic solution though given that the extra move makes it functionally impure. Either way the perf impact here seems like it'd ultimately be negligible so I'm kind of splitting hairs here